### ISSUE TITLE
Session ID

### ISSUE THREAD

**Issue by bakerac4** [Psychoanalytic-Electronic-Publishing/OpenPubArchive-Content-Server#55]
@nrshapiro ,

Im working on adding the PADS authentication. I seem to have everything working correctly, however no matter who Im logged in with I'm not allowed to read full documents.

I get the following returned in the JSON

```
accessLimited: true
accessLimitedDescription: "The full content of the document is available to subscribers. "
accessLimitedPubLink: null
accessLimitedReason: "This is a summary excerpt from the full text of the document. The full content of the document is available to subscribers. "
```

I just wanted to confirm that the session header I should be sending is `client_session` and not `client-session`. Otherwise - hoping you have some guidance on why Im seeing this?

**Comment by nrshapiro** [comment]
You need to use dashes instead of underscores for header variables.  I believe that's also the way I specified it in all the doc; the only one that was not specified with dashes was the api-key variables, but now they are dashes too, since apparently beanstalk does not let _ variables through.

Ah, I see how this might be a little confusing in the interactive server docs:

"Requires API key and client_id in the header (use 'client-id' in call header)."

I'll get rid of the mapped variable name reference part but the header requirement is client-id and client-session)

In short, all of the header variables use dashes.

**Comment by bakerac4** [comment]
@nrshapiro ok, I can switch that. However when I send in dashes all endpoints seem to return 500 errors. Could you take a look at that?

**Comment by nrshapiro** [comment]
I see...somewhere in the chain, client-session is being stripped out of the headers before I can read it.  I'm looking into it.

**Comment by nrshapiro** [comment]
@bakerac4 

Sorry, it's been a busy day.  And I didn't discover the real problem until I started running my Postman suite, and then not until I was able to fix the pretest calls to PaDS authenticate (which had changed, now requires Content-Type in headers, even though there's no body.)

I am pretty sure it should be ok now.  I pushed it to your staging.

**Comment by nrshapiro** [comment]
My tests verify it's ok.
